components/ILIAS/User/classes/class.ilObjUserGUI.php

```diff
@@ -1896,6 +1896,11 @@ public static function _goto(string $a_target): void
             $ilCtrl->redirectByClass(['ilStartUpGUI', 'ilPasswordAssistanceGUI'], 'showUsernameAssistanceForm');
         } elseif ('pwassist' == $a_target) {
             $ilCtrl->redirectByClass(['ilStartUpGUI', 'ilPasswordAssistanceGUI'], '');
+        } elseif ('login' === $a_target) {
+            $ilCtrl->redirectToURL('/login.php?' . http_build_query([
```

Comment:
Do we need a `cmd=foce_login` query parameter here? IMO there are a few occurrences in our code base, where this parameter is appended to ensure the display of the login page, even it the user/initialisation state is "undefined".